src/components/notecard-component/NoteCard.jsx

```diff
@@ -1,26 +1,37 @@
 import axios from "axios";
-import React from "react";
+import React,{useState} from "react";
 import { useNotes } from "../../context/useNotes";
 import "./NoteCard.css";
 import { addOrRemoveFromArchive } from "../../utils/notes-util/addOrRemoveFromArchive";
 import { addToTrash } from "../../utils/trash-util/addToTrash";
+import { HexColorPicker } from "react-colorful";
+import { changeColor } from "../../utils/notes-util/changeColor";
```

Comment:
You can create a js file to export all the util js files and import from there wherever you need when you refactor the code.

Reply:
Got it.